### PR TITLE
04-single-hop-6lowpan-icmp/task04: reconnect terminal after long test

### PR DESCRIPTION
**Pull request by miri64**
Task 4.4 fails on the CI, since the Github servers appear to silently drop the TCP connection to the node after a while (the pinged node is not accessed for the duration of the ping). By just starting the terminal anew, this is fixed. I will start a new run, once I merged this PR into #174.

This requires https://github.com/RIOT-OS/riotctrl/pull/6 to work.

### CODE REVIEW

**Comment by miri64** [comment]
Ok... even that does not seem to fix it. :-( And I'm honestly out of ideas what is going wrong. When running the script locally (but on IoT-LAB nodes) both variants (the one in master and this) work, but not in the GitHub actions. [Here](https://github.com/miri64/RIOT/runs/932162276) is a variant on what is happening on master (with slight modifications to that include #173, #178, #180, and an early version of #182) and [here](https://github.com/miri64/RIOT/actions/runs/190682598) a version corresponding with this PR (even having some more waiting times between disconnect and connect to the terminal build in). Both have a decreased ping count compared to the original task 4.4, so I did not have to wait the full 15 min—when decreasing too much by the way the bug does not show up. Additionally, both contain the logs as generated by #182+RIOT-OS/RIOT#14664, so basically what a user would see when typing `make term` for the board. In both `pktbuf` just doesn't show up.

@aabadie, @fjmolinas do you maybe have some ideas? The logs for the respective experiments (Exp-ID 225501 for the 932162276/master run, Exp-ID 225648 for the 190682598/this PR run) don't really offer much more information, sadly.

<details><summary><a href="https://www.iot-lab.info/testbed/experiments/225501">225501</a> (master-equivalent)</summary>

```
$ tail -n +1 .iot-lab/225501/log/*.log
==> .iot-lab/225501/log/m3_10.log <==
2020-07-31 15:50:52,485 :: INFO :: Start experiment: rici-225501
2020-07-31 15:50:52,498 :: INFO :: Control node : Starting of control node serial interface
2020-07-31 15:50:52,509 :: INFO :: Control node : reset
2020-07-31 15:50:52,521 :: INFO :: Reset Control Node
2020-07-31 15:50:55,630 :: INFO :: Control node : start power of open node
2020-07-31 15:50:55,690 :: INFO :: Node OpenOCD: Setup of openocd node
2020-07-31 15:50:58,730 :: INFO :: Node OpenOCD: flash of openocd node
2020-07-31 15:50:58,745 :: INFO :: Flash firmware on OpenOCD: /usr/lib/python2.7/site-packages/gateway_code/static/m3_idle.elf
2020-07-31 15:51:01,544 :: INFO :: Control node : Start experiment
2020-07-31 15:51:01,602 :: INFO :: Control Node set time delay: 1068 us
2020-07-31 15:51:01,619 :: INFO :: Control node : profile configuration
2020-07-31 15:51:01,623 :: INFO :: Configure profile on Control Node
2020-07-31 15:51:01,666 :: INFO :: Start experiment succeeded
2020-07-31 15:52:02,433 :: INFO :: Gateway Manager : Flash of node
2020-07-31 15:52:02,442 :: INFO :: Flash firmware on open node: /tmp/tmpwLrSt8--firmware4491895237730011215-gnrc_networking.bin
2020-07-31 15:52:02,453 :: INFO :: Node OpenOCD: flash of openocd node
2020-07-31 15:52:02,460 :: INFO :: Flash firmware on OpenOCD: /tmp/tmpwLrSt8--firmware4491895237730011215-gnrc_networking.bin
2020-07-31 16:00:53,204 :: INFO :: Gateway Manager : Stop experiment
2020-07-31 16:00:53,226 :: INFO :: Stop experiment
2020-07-31 16:00:53,244 :: INFO :: Control node : stop of the experiment
2020-07-31 16:00:53,266 :: INFO :: Control node : profile configuration
2020-07-31 16:00:53,282 :: INFO :: Configure profile on Control Node
2020-07-31 16:00:53,430 :: INFO :: Control node : start power of open node
2020-07-31 16:00:53,456 :: INFO :: Node OpenOCD: teardown of openocd node
2020-07-31 16:00:56,491 :: INFO :: openocd node debugger stop
2020-07-31 16:00:57,146 :: INFO :: Node OpenOCD: flash of openocd node
2020-07-31 16:00:57,162 :: INFO :: Flash firmware on OpenOCD: /usr/lib/python2.7/site-packages/gateway_code/static/m3_idle.elf
2020-07-31 16:00:59,782 :: INFO :: Control node : Stop control node serial interface
2020-07-31 16:00:59,791 :: INFO :: Control node : stop power of open node
2020-07-31 16:00:59,847 :: INFO :: Control node : reset
2020-07-31 16:00:59,854 :: INFO :: Reset Control Node
2020-07-31 16:01:02,754 :: INFO :: Stop experiment succeeded

==> .iot-lab/225501/log/samr21_10.log <==
2020-07-31 15:50:52,485 :: INFO :: Start experiment: rici-225501
2020-07-31 15:50:52,498 :: INFO :: Control node : Starting of control node serial interface
2020-07-31 15:50:52,509 :: INFO :: Control node : reset
2020-07-31 15:50:52,521 :: INFO :: Reset Control Node
2020-07-31 15:50:55,637 :: INFO :: Control node : start power of open node
2020-07-31 15:50:55,693 :: INFO :: Node EDBG: Setup of EDBG node
2020-07-31 15:50:57,637 :: INFO :: Node EDBG: flash of edbg node
2020-07-31 15:50:57,644 :: INFO :: Flash firmware on SAMR21: /usr/lib/python2.7/site-packages/gateway_code/static/samr21_idle.elf
2020-07-31 15:50:57,656 :: INFO :: Creating bin file from /usr/lib/python2.7/site-packages/gateway_code/static/samr21_idle.elf
2020-07-31 15:50:57,667 :: INFO :: Created bin file in /tmp/tmp1vDfjF.bin
2020-07-31 15:50:57,688 :: INFO :: {'args': ['objcopy', '-I', 'elf32-big', '-O', 'binary', '/usr/lib/python2.7/site-packages/gateway_code/static/samr21_idle.elf', '/tmp/tmp1vDfjF.bin'], 'stderr': <open file '/dev/null', mode 'w' at 0xb61f46a8>, 'stdout': <open file '/dev/null', mode 'w' at 0xb61f46a8>}
2020-07-31 15:50:57,763 :: INFO :: To bin conversion ret value : 0
2020-07-31 15:50:57,772 :: INFO :: {'args': ['/usr/bin/edbg', '-t', 'atmel_cm0p', '-b', '-e', '-v', '-p', '-f', '/tmp/tmp1vDfjF.bin', '-o', '0x0'], 'stderr': <open file '/dev/null', mode 'w' at 0xb61f46a8>, 'stdout': <open file '/dev/null', mode 'w' at 0xb61f46a8>}
2020-07-31 15:50:59,306 :: INFO :: Flashing ret value : 0
2020-07-31 15:50:59,397 :: INFO :: Control node : Start experiment
2020-07-31 15:50:59,429 :: INFO :: Control Node set time delay: 1007 us
2020-07-31 15:50:59,438 :: INFO :: Control node : profile configuration
2020-07-31 15:50:59,441 :: INFO :: Configure profile on Control Node
2020-07-31 15:50:59,484 :: INFO :: Start experiment succeeded
2020-07-31 15:51:26,427 :: INFO :: Gateway Manager : Flash of node
2020-07-31 15:51:26,437 :: INFO :: Flash firmware on open node: /tmp/tmpq0espf--firmware7380581138446092835-gnrc_networking.bin
2020-07-31 15:51:26,448 :: INFO :: Node EDBG: flash of edbg node
2020-07-31 15:51:26,454 :: INFO :: Flash firmware on SAMR21: /tmp/tmpq0espf--firmware7380581138446092835-gnrc_networking.bin
2020-07-31 15:51:26,482 :: INFO :: {'args': ['/usr/bin/edbg', '-t', 'atmel_cm0p', '-b', '-e', '-v', '-p', '-f', '/tmp/tmpq0espf--firmware7380581138446092835-gnrc_networking.bin', '-o', '0x0'], 'stderr': <open file '/dev/null', mode 'w' at 0xb61f46a8>, 'stdout': <open file '/dev/null', mode 'w' at 0xb61f46a8>}
2020-07-31 15:51:40,272 :: INFO :: Flashing ret value : 0
2020-07-31 16:00:53,203 :: INFO :: Gateway Manager : Stop experiment
2020-07-31 16:00:53,223 :: INFO :: Stop experiment
2020-07-31 16:00:53,241 :: INFO :: Control node : stop of the experiment
2020-07-31 16:00:53,255 :: INFO :: Control node : profile configuration
2020-07-31 16:00:53,269 :: INFO :: Configure profile on Control Node
2020-07-31 16:00:53,405 :: INFO :: Control node : start power of open node
2020-07-31 16:00:53,456 :: INFO :: Node EDBG: teardown of EDBG node
2020-07-31 16:00:56,493 :: INFO :: SAMR21 Node debugger stop
2020-07-31 16:00:57,178 :: INFO :: Node EDBG: flash of edbg node
2020-07-31 16:00:57,193 :: INFO :: Flash firmware on SAMR21: /usr/lib/python2.7/site-packages/gateway_code/static/samr21_idle.elf
2020-07-31 16:00:57,219 :: INFO :: Creating bin file from /usr/lib/python2.7/site-packages/gateway_code/static/samr21_idle.elf
2020-07-31 16:00:57,246 :: INFO :: Created bin file in /tmp/tmpnjkK9f.bin
2020-07-31 16:00:57,293 :: INFO :: {'args': ['objcopy', '-I', 'elf32-big', '-O', 'binary', '/usr/lib/python2.7/site-packages/gateway_code/static/samr21_idle.elf', '/tmp/tmpnjkK9f.bin'], 'stderr': <open file '/dev/null', mode 'w' at 0xb61f46a8>, 'stdout': <open file '/dev/null', mode 'w' at 0xb61f46a8>}
2020-07-31 16:00:57,517 :: INFO :: To bin conversion ret value : 0
2020-07-31 16:00:57,526 :: INFO :: {'args': ['/usr/bin/edbg', '-t', 'atmel_cm0p', '-b', '-e', '-v', '-p', '-f', '/tmp/tmpnjkK9f.bin', '-o', '0x0'], 'stderr': <open file '/dev/null', mode 'w' at 0xb61f46a8>, 'stdout': <open file '/dev/null', mode 'w' at 0xb61f46a8>}
2020-07-31 16:00:59,210 :: INFO :: Flashing ret value : 0
2020-07-31 16:00:59,217 :: INFO :: Control node : Stop control node serial interface
2020-07-31 16:00:59,220 :: INFO :: Control node : stop power of open node
2020-07-31 16:00:59,253 :: INFO :: Control node : reset
2020-07-31 16:00:59,257 :: INFO :: Reset Control Node
2020-07-31 16:01:02,254 :: INFO :: Stop experiment succeeded
```

</details>

<details><summary><a href="hhttps://www.iot-lab.info/testbed/experiments/225648">225648</a> (equivalent of this PR)</summary>

```
$ tail -n +1 .iot-lab/225648/log/*.log
==> .iot-lab/225648/log/m3_10.log <==
2020-08-01 02:09:52,917 :: INFO :: Start experiment: rici-225648
2020-08-01 02:09:52,930 :: INFO :: Control node : Starting of control node serial interface
2020-08-01 02:09:52,941 :: INFO :: Control node : reset
2020-08-01 02:09:52,953 :: INFO :: Reset Control Node
2020-08-01 02:09:56,111 :: INFO :: Control node : start power of open node
2020-08-01 02:09:56,170 :: INFO :: Node OpenOCD: Setup of openocd node
2020-08-01 02:09:59,213 :: INFO :: Node OpenOCD: flash of openocd node
2020-08-01 02:09:59,228 :: INFO :: Flash firmware on OpenOCD: /usr/lib/python2.7/site-packages/gateway_code/static/m3_idle.elf
2020-08-01 02:10:01,874 :: INFO :: Control node : Start experiment
2020-08-01 02:10:01,946 :: INFO :: Control Node set time delay: 1618 us
2020-08-01 02:10:01,968 :: INFO :: Control node : profile configuration
2020-08-01 02:10:01,976 :: INFO :: Configure profile on Control Node
2020-08-01 02:10:02,013 :: INFO :: Start experiment succeeded
2020-08-01 02:11:13,360 :: INFO :: Gateway Manager : Flash of node
2020-08-01 02:11:13,364 :: INFO :: Flash firmware on open node: /tmp/tmpHkoFWl--firmware3341625996566210030-gnrc_networking.bin
2020-08-01 02:11:13,370 :: INFO :: Node OpenOCD: flash of openocd node
2020-08-01 02:11:13,373 :: INFO :: Flash firmware on OpenOCD: /tmp/tmpHkoFWl--firmware3341625996566210030-gnrc_networking.bin
2020-08-01 02:16:58,508 :: INFO :: Gateway Manager : Stop experiment
2020-08-01 02:16:58,520 :: INFO :: Stop experiment
2020-08-01 02:16:58,529 :: INFO :: Control node : stop of the experiment
2020-08-01 02:16:58,537 :: INFO :: Control node : profile configuration
2020-08-01 02:16:58,544 :: INFO :: Configure profile on Control Node
2020-08-01 02:16:58,621 :: INFO :: Control node : start power of open node
2020-08-01 02:16:58,665 :: INFO :: Node OpenOCD: teardown of openocd node
2020-08-01 02:17:01,708 :: INFO :: openocd node debugger stop
2020-08-01 02:17:02,356 :: INFO :: Node OpenOCD: flash of openocd node
2020-08-01 02:17:02,371 :: INFO :: Flash firmware on OpenOCD: /usr/lib/python2.7/site-packages/gateway_code/static/m3_idle.elf
2020-08-01 02:17:05,003 :: INFO :: Control node : Stop control node serial interface
2020-08-01 02:17:05,012 :: INFO :: Control node : stop power of open node
2020-08-01 02:17:05,066 :: INFO :: Control node : reset
2020-08-01 02:17:05,073 :: INFO :: Reset Control Node
2020-08-01 02:17:07,973 :: INFO :: Stop experiment succeeded

==> .iot-lab/225648/log/samr21_10.log <==
2020-08-01 02:09:52,917 :: INFO :: Start experiment: rici-225648
2020-08-01 02:09:52,929 :: INFO :: Control node : Starting of control node serial interface
2020-08-01 02:09:52,941 :: INFO :: Control node : reset
2020-08-01 02:09:52,952 :: INFO :: Reset Control Node
2020-08-01 02:09:56,065 :: INFO :: Control node : start power of open node
2020-08-01 02:09:56,123 :: INFO :: Node EDBG: Setup of EDBG node
2020-08-01 02:09:57,964 :: INFO :: Node EDBG: flash of edbg node
2020-08-01 02:09:57,969 :: INFO :: Flash firmware on SAMR21: /usr/lib/python2.7/site-packages/gateway_code/static/samr21_idle.elf
2020-08-01 02:09:57,975 :: INFO :: Creating bin file from /usr/lib/python2.7/site-packages/gateway_code/static/samr21_idle.elf
2020-08-01 02:09:57,981 :: INFO :: Created bin file in /tmp/tmp4Z8k7h.bin
2020-08-01 02:09:57,992 :: INFO :: {'args': ['objcopy', '-I', 'elf32-big', '-O', 'binary', '/usr/lib/python2.7/site-packages/gateway_code/static/samr21_idle.elf', '/tmp/tmp4Z8k7h.bin'], 'stderr': <open file '/dev/null', mode 'w' at 0xb61f46a8>, 'stdout': <open file '/dev/null', mode 'w' at 0xb61f46a8>}
2020-08-01 02:09:58,089 :: INFO :: To bin conversion ret value : 0
2020-08-01 02:09:58,099 :: INFO :: {'args': ['/usr/bin/edbg', '-t', 'atmel_cm0p', '-b', '-e', '-v', '-p', '-f', '/tmp/tmp4Z8k7h.bin', '-o', '0x0'], 'stderr': <open file '/dev/null', mode 'w' at 0xb61f46a8>, 'stdout': <open file '/dev/null', mode 'w' at 0xb61f46a8>}
2020-08-01 02:09:59,626 :: INFO :: Flashing ret value : 0
2020-08-01 02:09:59,708 :: INFO :: Control node : Start experiment
2020-08-01 02:09:59,746 :: INFO :: Control Node set time delay: 1373 us
2020-08-01 02:09:59,755 :: INFO :: Control node : profile configuration
2020-08-01 02:09:59,758 :: INFO :: Configure profile on Control Node
2020-08-01 02:09:59,803 :: INFO :: Start experiment succeeded
2020-08-01 02:10:31,760 :: INFO :: Gateway Manager : Flash of node
2020-08-01 02:10:31,770 :: INFO :: Flash firmware on open node: /tmp/tmpvIF7cQ--firmware5605658986283091797-gnrc_networking.bin
2020-08-01 02:10:31,781 :: INFO :: Node EDBG: flash of edbg node
2020-08-01 02:10:31,788 :: INFO :: Flash firmware on SAMR21: /tmp/tmpvIF7cQ--firmware5605658986283091797-gnrc_networking.bin
2020-08-01 02:10:31,816 :: INFO :: {'args': ['/usr/bin/edbg', '-t', 'atmel_cm0p', '-b', '-e', '-v', '-p', '-f', '/tmp/tmpvIF7cQ--firmware5605658986283091797-gnrc_networking.bin', '-o', '0x0'], 'stderr': <open file '/dev/null', mode 'w' at 0xb61f46a8>, 'stdout': <open file '/dev/null', mode 'w' at 0xb61f46a8>}
2020-08-01 02:10:45,856 :: INFO :: Flashing ret value : 0
2020-08-01 02:16:58,539 :: INFO :: Gateway Manager : Stop experiment
2020-08-01 02:16:58,559 :: INFO :: Stop experiment
2020-08-01 02:16:58,574 :: INFO :: Control node : stop of the experiment
2020-08-01 02:16:58,595 :: INFO :: Control node : profile configuration
2020-08-01 02:16:58,609 :: INFO :: Configure profile on Control Node
2020-08-01 02:16:58,740 :: INFO :: Control node : start power of open node
2020-08-01 02:16:58,795 :: INFO :: Node EDBG: teardown of EDBG node
2020-08-01 02:17:01,831 :: INFO :: SAMR21 Node debugger stop
2020-08-01 02:17:02,510 :: INFO :: Node EDBG: flash of edbg node
2020-08-01 02:17:02,517 :: INFO :: Flash firmware on SAMR21: /usr/lib/python2.7/site-packages/gateway_code/static/samr21_idle.elf
2020-08-01 02:17:02,530 :: INFO :: Creating bin file from /usr/lib/python2.7/site-packages/gateway_code/static/samr21_idle.elf
2020-08-01 02:17:02,545 :: INFO :: Created bin file in /tmp/tmpPmkUkT.bin
2020-08-01 02:17:02,567 :: INFO :: {'args': ['objcopy', '-I', 'elf32-big', '-O', 'binary', '/usr/lib/python2.7/site-packages/gateway_code/static/samr21_idle.elf', '/tmp/tmpPmkUkT.bin'], 'stderr': <open file '/dev/null', mode 'w' at 0xb61f46a8>, 'stdout': <open file '/dev/null', mode 'w' at 0xb61f46a8>}
2020-08-01 02:17:02,687 :: INFO :: To bin conversion ret value : 0
2020-08-01 02:17:02,707 :: INFO :: {'args': ['/usr/bin/edbg', '-t', 'atmel_cm0p', '-b', '-e', '-v', '-p', '-f', '/tmp/tmpPmkUkT.bin', '-o', '0x0'], 'stderr': <open file '/dev/null', mode 'w' at 0xb61f46a8>, 'stdout': <open file '/dev/null', mode 'w' at 0xb61f46a8>}
2020-08-01 02:17:04,312 :: INFO :: Flashing ret value : 0
2020-08-01 02:17:04,318 :: INFO :: Control node : Stop control node serial interface
2020-08-01 02:17:04,322 :: INFO :: Control node : stop power of open node
2020-08-01 02:17:04,355 :: INFO :: Control node : reset
2020-08-01 02:17:04,359 :: INFO :: Reset Control Node
2020-08-01 02:17:07,231 :: INFO :: Stop experiment succeeded
```

</details>

**Comment by miri64** [comment]
Rebased to current master

**Comment by fjmolinas** [comment]
Is there any difference in the logs I should look at carefully?

**Comment by miri64** [comment]
> Is there any difference in the logs I should look at carefully?

No, as I said: I did not find the reason for this bug to happen and this PR does not fix it. That is why #183 exists.

**Comment by miri64** [comment]
What I was asking you and @aabadie about was if you know about any other way to debug the problem, because with the ways I know, I did not came to conclusive results.

**Comment by fjmolinas** [comment]
> No, as I said: I did not find the reason for this bug to happen and this PR does not fix it. That is why #183 exists.

Thanks for the clarification!

**Comment by miri64** [comment]
>  because with the ways I know, I did not came to conclusive results.

(the ways I used to debug the problems are what I describe in https://github.com/RIOT-OS/Release-Specs/pull/181#issuecomment-667517820 ;-))

**Comment by miri64** [comment]
Rebased to current master. IMHO we should merge #183 for now and then see here if we find a way to fix it. The automated release tests are no value to us if they are failing all the time.

**Comment by fjmolinas** [comment]
Did not mean to close, the issue is still present.

File: 04-single-hop-6lowpan-icmp/test_spec04.py [hunk 1]
```diff
@@ -113,6 +113,11 @@ def test_task04(riot_ctrl):
                 count=10000, interval=100, packet_size=100)
     assert res['stats']['packet_loss'] < 10
 
+    # enforce reconnect to pinged's terminal as connection to it in the IoT-LAB
+    # sometimes get's lost silently in the CI after the 15 min of pinging
+    pinged.stop_term()
```
Comment (miri64) [inline 04-single-hop-6lowpan-icmp/test_spec04.py:118]: From offline discussion with @fjmolinas: we might get this working by moving this above l112, as the connection from the IoT-LAB side might still be perceived as open after this and this the `start_term()` fails silently. By moving this above the ping there is enough time for the connection to time out properly.

**Comment by miri64** [comment]
Rebased and applied https://github.com/RIOT-OS/Release-Specs/pull/181#discussion_r464995248.

File: 04-single-hop-6lowpan-icmp/test_spec04.py (hunk 1 above)
Comment (miri64) [reply 04-single-hop-6lowpan-icmp/test_spec04.py]: I started [a run](https://github.com/miri64/RIOT/runs/951201329) containing [this PR](https://github.com/miri64/RIOT/runs/951201329?check_suite_focus=true#step:3:184). Let's see if this theory turns out true

**Comment by miri64** [comment]
Appearently the fix proposed in https://github.com/RIOT-OS/Release-Specs/pull/181#pullrequestreview-460742061 works. See this action: https://github.com/miri64/RIOT/actions/runs/196824389

**Comment by miri64** [comment]
Rebased to current master. @aabadie @fjmolinas as this fixes the test in the CI, I propose to merge this now. However, just to be safe I start another run with the most current version of this PR and master.

**Comment by miri64** [comment]
> However, just to be safe I start another run with the most current version of this PR and master.

See https://github.com/miri64/RIOT/actions/runs/197554736

**Comment by fjmolinas** [comment]
> Rebased to current master. @aabadie @fjmolinas as this fixes the test in the CI, I propose to merge this now. However, just to be safe I start another run with the most current version of this PR and master.

That is fine with me, but since there is still an issue that we are simply working around, can we open an Issue with pretty much the findings of this PR?

**Comment by miri64** [comment]
See https://github.com/RIOT-OS/Release-Specs/issues/189. I also added a comment referring to that issue to the test.

**Comment by fjmolinas** [comment]
> See #189. I also added a comment referring to that issue to the test.

Will wait for the new run to finish and ACK

**Comment by miri64** [comment]
> > See #189. I also added a comment referring to that issue to the test.
> 
> Will wait for the new run to finish and ACK

I did not start a new run after https://github.com/miri64/RIOT/actions/runs/197554736, since I don't think the comment will make much difference (and the syntax is checked by `flake8` and `pylint` here)

**Comment by fjmolinas** [comment]
> I did not start a new run after https://github.com/miri64/RIOT/actions/runs/197554736, since I don't think the comment will make much difference (and the syntax is checked by flake8 and pylint here)

it seems to have on its own then...

**Comment by miri64** [comment]
> > I did not start a new run after https://github.com/miri64/RIOT/actions/runs/197554736, since I don't think the comment will make much difference (and the syntax is checked by flake8 and pylint here)
> 
> it seems to have on its own then...
Huh? How do you mean? Currently I only see that run running here https://github.com/miri64/RIOT/actions and that is running since I started it about 2h ago. Not since I squashed the comment in.

**Comment by fjmolinas** [comment]
> Huh? How do you mean? Currently I only see that run running here https://github.com/miri64/RIOT/actions and that is running since I started it about 2h ago. Not since I squashed the comment in.

That was the run I meant :)

**Comment by miri64** [comment]
But it did nothing on its own, so I was confused by your statement ^^

**Comment by fjmolinas** [comment]
> But it did nothing on its own, so I was confused by your statement ^^

I should have just said "currrent" run instead of "new" run and you wouldn't have been confused, which confused me, which confused you, which confused you. Anyway glad we are on the same page now.

**Comment by fjmolinas** [comment]
Seems like flashing failed on one of the runs, not sure why but unrelated to this PR

```
04-single-hop-6lowpan-icmp/test_spec04.py::test_task01[nodes0] PASSED    [  5%]
04-single-hop-6lowpan-icmp/test_spec04.py::test_task02[nodes0] PASSED    [ 11%]
04-single-hop-6lowpan-icmp/test_spec04.py::test_task03[nodes0] PASSED    [ 17%]
04-single-hop-6lowpan-icmp/test_spec04.py::test_task04[nodes0] PASSED    [ 23%]
04-single-hop-6lowpan-icmp/test_spec04.py::test_task09[nodes0] PASSED    [ 29%]
04-single-hop-6lowpan-icmp/test_spec04.py::test_task10[nodes0] PASSED    [ 35%]
05-single-hop-route/test_spec05.py::test_task02[nodes0] PASSED           [ 41%]
06-single-hop-udp/test_spec06.py::test_task01[nodes0] PASSED             [ 47%]
06-single-hop-udp/test_spec06.py::test_task02[nodes0] PASSED             [ 52%]
06-single-hop-udp/test_spec06.py::test_task04[nodes0] PASSED             [ 58%]
06-single-hop-udp/test_spec06.py::test_task06[nodes0] PASSED             [ 64%]
07-multi-hop/test_spec07.py::test_task01[nodes0] PASSED                  [ 70%]
07-multi-hop/test_spec07.py::test_task02[nodes0] PASSED                  [ 76%]
07-multi-hop/test_spec07.py::test_task03[nodes0] ERROR                   [ 82%]
07-multi-hop/test_spec07.py::test_task04[nodes0] PASSED                  [ 88%]
07-multi-hop/test_spec07.py::test_task05[nodes0] XFAIL                   [ 94%]
08-interop/test_spec08.py::test_task08[nodes0] PASSED                    [100%]
```

```
>               raise CalledProcessError(retcode, process.args,
                                         output=stdout, stderr=stderr)
E               subprocess.CalledProcessError: Command '['make', '--no-print-directory', '-C', '/home/runner/work/RIOT/RIOT/RIOT/tests/gnrc_udp', 'flash']' returned non-zero exit status 2.
```

**Comment by miri64** [comment]
It's due to a [Bad Gateway error](https://github.com/miri64/RIOT/runs/953312829?check_suite_focus=true#step:11:468) this seems unrelated an an error on the IoT-LAB side (probably short gateway outage on one of the control nodes?)

**Pull request merged**